source/release-notes/mongoid-8.0.txt

```diff
@@ -824,3 +824,14 @@ Support for individually caching criteria objects has been dropped in Mongoid 8.
 
 In order to get caching functionality, enable the Mongoid Query Cache. See the
 section on :ref:`Query Cache <query-cache>` for more details.
+
+New Caching Behavior of ``<association>_ids`` Methods
+-----------------------------------------------------
+
+Mongoid 8 uses in-memory collection to satisfy the query from calling an
+``<association>_ids`` method if the association is already loaded into memory
```

Comment:
S: I think giving an example of an ``<association>_ids`` method would be helpful, since it's kinda unclear what that means. Also, I noticed these release notes use both Mongoid 8 / Mongoid 8.0, but 8.0 aligns with our usual syntax for driver versions.

With those suggestions in mind, maybe this could say something like:

```suggestion
If you call an ``<association>_ids`` method, such as ``member_ids``, on an association previously loaded into memory, Mongoid 8.0 uses the in-memory collection to satisfy the query
```